jqm-all/jqm-model/pom.xml

```diff
@@ -18,5 +18,11 @@
 			<artifactId>slf4j-api</artifactId>
 			<version>${slf4j.version}</version>
 		</dependency>
+
+		<dependency>
```

Comment:
The model should not have a dependency on this lib - it is used by our APIs which in turn may be used in a lot of contexts, so we should keep dependencies minimal.